x-pack/filebeat/input/azureeventhub/eph.go

```diff
@@ -74,3 +86,15 @@ func (a *azureInput) runWithEPH() error {
 	}
 	return nil
 }
+
+func getAzureEnvironment(overrideResManager string) (azure.Environment, error) {
```

Comment:
Would it make sense to add some unit test for this method?

Reply:
added a test